File: pb/src/main/java/org/sqlbuilder/sl/VnClassRoleResolver.java
package org.sqlbuilder.sl;

import org.sqlbuilder.common.DeSerialize;
import org.sqlbuilder.common.Resolver;
import org.sqlbuilder2.ser.Pair;
import org.sqlbuilder2.ser.Triplet;

import java.io.File;
import java.io.IOException;

public class VnClassRoleResolver extends Resolver<Pair<String,String>, Triplet<Integer,Integer,Integer>>
{
	public VnClassRoleResolver(final String ser) throws IOException, ClassNotFoundException
	{
		super(DeSerialize.deserialize(new File(ser)));
	}
}

File: pb31/src/main/java/org/sqlbuilder/pb/VnClassRoleResolver.kt
package org.sqlbuilder.pb

import org.sqlbuilder.common.DeSerialize
import org.sqlbuilder.common.Resolver
import java.io.File

class VnClassRoleResolver(ser: String) : Resolver<Pair<String, String>, Triple<Int, Int, Int>>(DeSerialize.deserialize<Map<Pair<String, String>, Triple<Int, Int, Int>>>(File(ser)))
